files/ks/rhel8.cfg.pkrtpl.hcl

```diff
@@ -25,7 +25,7 @@ timezone America/Los_Angeles --utc
 #network --bootproto dhcp --noipv6
 
 # B.5. Kickstart commands for handling storage
-autopart --fstype=xfs --nohome --type=lvm --noswap
+autopart --fstype=xfs --nohome --type=lvm ${swap}
```

Comment:
@sgurnick, @jhriv:
Did we change our mind to use swap partitions instead of swapfiles?

I'm not familiar enough with `autopart`. You'll have to test this out. I've only used `part` in ks files
```
part /boot --fstype="xfs" --ondisk=sda --size=1024 --asprimary
part swap --fstype="swap" --ondisk=sda --size=1024 --asprimary
part / --fstype xfs --size 1 --grow --asprimary 
```

Reply:
`autopart` does similar, but is more device-agnostic. As it will also make a separate /home partition if the drive is large enough (>50GB) so we prevent that and come up with a similar partition scheme. This kickstart file is also using LVM instead of raw partitions

I believe kickstart will size the swap partition based on allocated memory, but I can't find documentation to cite, and I don't have any tests to conform or deny.

For existing systems, we'd still need to add a swap file to enable swap, but that would be for a limited number of systems